helm/vitalam-identity-service/Chart.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: v2
+name: vitalam-identity-service
+appVersion: '0.0.1'
+description: A Helm chart for vitalam-identity-service
+version: '0.0.1'
+type: application
+maintainers:
+  - name: digicatapult
+    email: opensource@digicatapult.org.uk
+    url: www.digicatapult.org.uk
+dependencies:
+  - name: postgresql
+    version: '^10.12.2'
+    repository: https://charts.bitnami.com/bitnami
+    condition: postgresql.enabled
```

Comment:
Odds are the `vitalam-node` should be a dependency here, but that should probably be a separate PR. Can you create an issue for that when this merges?